### PR TITLE
Fix ada normalizer

### PR DESCRIPTION
**Pull request by peter-mlabs**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,7 +2,14 @@
 
 This format is based on [Keep A Changelog](https://keepachangelog.com/en/1.0.0).
 
-## 2.5.0 -- 2020-09-26
+## 2.6.0 -- 2022-09-27
+
+### Fixed
```
Comment (kozross) [inline CHANGELOG.md:7]: 'Modified' is probably the section for this.

**Review by kozross — CHANGES_REQUESTED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,7 +2,16 @@
...
+* `normalizeValue` now has the correct behavior of:
+  * adding a 0 Ada entry to the underlying map if the 
+    entry would otherwise be missing.
+  * removing entires with the Ada currency symbol, but a non-Ada
```
Comment (kozross) [inline CHANGELOG.md:11]: 'entries'

**Review by kozross — CHANGES_REQUESTED** [review]

File: src/Plutarch/Context/Base.hs [hunk 3]
```diff
@@ -779,18 +779,42 @@ sortMap (AssocMap.toList -> m) =
...
+          Nothing -> valWith0Ada -- No Ada token present, add the 0 entry
+          Just adaAmount ->
+            {- Ada symbol and token present, strip out ada symbol
+               entires (which may contain non-Ada tokens and/or duplicates),
+               and add back in only the correct Ada asset class amount
+            -}
+            AssocMap.insert
+              adaSymbol
+              (AssocMap.fromList [(adaToken, adaAmount)])
+              ( AssocMap.fromList $
```
Comment (SeungheonOh) [inline src/Plutarch/Context/Base.hs:815]: I think `AssocMap.mapMaybeWithKey` can be used here. I'm fine with either way.

**Review by SeungheonOh — APPROVED** [review]
LGTM

**Review by kozross — APPROVED** [review]

**Review by kozross — CHANGES_REQUESTED** [review]
Cabal file needs higher version number.

**Review by SeungheonOh — APPROVED** [review]
LGTM

**Review by kozross — APPROVED** [review]

**Pull request merged**